scripts/spack_packages/raja_perf/package.py

```diff
@@ -217,10 +217,11 @@ def hostconfig(self, spec, prefix, py_site_pkgs_dir=None):
         gcc_name_regex = re.compile(".*gcc-name.*")
 
         using_toolchain = list(filter(gcc_toolchain_regex.match, spec.compiler_flags['cxxflags']))
+
         if(using_toolchain):
           gcc_toolchain_path = gcc_toolchain_regex.match(using_toolchain[0])
         using_gcc_name = list(filter(gcc_name_regex.match, spec.compiler_flags['cxxflags']))
-        compilers_using_toolchain = ["pgi", "xl", "icpc"]
+        compilers_using_toolchain = ["pgi", "xl", "icpc", "clang"]
```

Comment:
Do we still need PGI here?

Reply:
Or icpc? Lol

Reply:
Keep for now. We will take a pass through to re-assess and update compilers and versions after the rest of the CI changes are in.